packages/foreman/foreman/foreman.spec

```diff
@@ -485,75 +486,75 @@ Requires: %{name} = %{version}-%{release}
 Requires: %{scl}-runtime-assets >= 5
 Requires: %{scl}-runtime-assets < 6
 %endif
```

Comment:
Does this need the sale `%else` as in the header?

Reply:
Not following this question

Reply:
https://github.com/theforeman/foreman-packaging/pull/4192/files#diff-3399e5c03dc282964481f8f3ced9c141R251-R256

Reply:
Sorry, I still am not clear what you are asking for.

Reply:
I think Ewoud asks if we need to add

```
%else
Requires: nodejs-packaging
```
for non-SCL'ed builds

but I wonder when this is the case, because `scl` refers to *our* SCL, and we always build against TFM these days?

Reply:
Yes, that's what I meant.

Reply:
I suppose not if things are working already?

Reply:
probably, yeah. the diff just looked weird to me